packs/napalm/config.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
What's the distinction? I am basing this off of the other packs I saw in st2contrib, which all use `config.yaml`